### DISCUSSION TITLE
How to Contribute to software ?

### DISCUSSION THREAD

**Discussion by bilalfawadkhan** [geovens/gInk#198] [General]
Hi, Is there anyway I can contribute to the code , I am looking forward to fix bugs and implement new ideas as per issues request and ideas discussion. How ever I cant seem to find How to contribute tab and how to setup work environment for it. I would like help in setting a documentation also on how others can contribute to the code

**Comment by pubpub-zz** [comment]
for my fork https://github.com/pubpub-zz/ppInk
I'm using Visual Studio 2017 community version (I had some troubles with further versions but I'm not an expert=. you should be able to load the files through git (I personally use GitKraken) (easier to propose PR)
you may be able to get visual studio from there:
https://visualstudio.microsoft.com/fr/vs/older-downloads/

I've got some contributions on my forks and any help is welcomed 😉

**Reply by bilalfawadkhan** [reply]
Thanks for the Prompt response and information., I am impressed by your fork of Gink. Looking forward to it.

**Reply by pubpub-zz** [reply]
Do not hesitate to star it 😜
